src/uct/tcp/tcp_cm.c

```diff
@@ -501,7 +501,7 @@ uct_tcp_cm_verify_req_connected_ep(uct_tcp_ep_t *ep,
     /* copy iface_addr to the local variable to avoid potential unaligned access
      * when need to get the address of uct_tcp_cm_conn_req_pkt_t::iface_addr,
      * since uct_tcp_cm_conn_req_pkt_t is a packed structure */
-    struct sockaddr_storage pkt_addr = cm_req_pkt->iface_addr;
+    const struct sockaddr_storage pkt_addr = cm_req_pkt->iface_addr;
```

Comment:
```suggestion
    const struct sockaddr_storage *pkt_addr = &cm_req_pkt->iface_addr;
```

Reply:
@dmitrygx  this generating the following static check error. look like there is comment also above in the code.
`/__w/2/s/src/uct/tcp/tcp_cm.c:545:47: error: taking address of packed member of 'struct uct_tcp_cm_conn_req_pkt' may result in an unaligned pointer value [-Werror=address-of-packed-member] <--[gcc`

Reply:
ah, I see. It was added by me in #5475